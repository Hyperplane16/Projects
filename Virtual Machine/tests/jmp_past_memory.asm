FUNC LABEL 0
    MOV REG 0 VAL 254
    MOV REG 1 VAL 100
    ADD REG 7 REG 0
    PRINT REG 0
    RET
