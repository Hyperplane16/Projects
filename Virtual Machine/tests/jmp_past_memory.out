PC references non-existent instruction in program memory.
1